atlas_densities/densities/mtype_densities_from_map/utils.py

```diff
@@ -32,6 +37,11 @@ def check_probability_map_sanity(probability_map: "pd.DataFrame") -> None:
             "The sum of each row is not 1.0. Consider renormalizing your input data frame with:\n"
             "df = df.div(np.sum(df, axis=1), axis=0)"
         )
+    if not {synapse_class for _, _, synapse_class in probability_map.index} <= SYNAPSE_CLASSES:
```

Comment:
what about something like:
```
if set(probability_map.index.get_level_values('synapse_class')) != SYNAPSE_CLASSES:
```

Reply:
I will stick with the inequality though

Reply:
sure, any particular reason?

Reply:
if the probability map only has `EXC` for example, it will not be equal to `{EXC, INH}` but it is still valid probability map